source/SkiaSharp.Views.Uno/SkiaSharp.Views.Uno.Wasm/SKXamlCanvas.Wasm.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System;
 using System.Runtime.InteropServices;
 using Uno.Foundation;
+using Uno.UI.Runtime.WebAssembly;
 using Windows.UI.Xaml;
 
 namespace SkiaSharp.Views.UWP
 {
+	[HtmlElement("canvas")]
 	public partial class SKXamlCanvas : FrameworkElement
 	{
-		private IntPtr pixels;
+		private byte[] pixels;
```

Comment:
Bringing the comments forward from the previous PR:

@mattleibow: What is the reason for switching from the Marshal to the managed array? I have no problem just wondering the reason.

@jeromelaban: This is for two reasons. The first is because the array needs to be memset to zero, and the second is looping in managed code can be quite slow with the interpreter, even when using pointers. Pinning the managed array and providing it to Skia is very fast.